tools/az-prom-rules-converter/test-files/template-example.json

```diff
@@ -65,7 +65,7 @@
                   },
                   {
                       "alert": "KubePodNotReady",
-                      "expression": "sum by (namespace, pod, cluster) (  max by(namespace, pod, cluster) (    kube_pod_status_phase{job=\"kube-state-metrics\", phase=~\"Pending|Unknown\"}  ) * on(namespace, pod, cluster) group_left(owner_kind) topk by(namespace, pod, cluster) (    1, max by(namespace, pod, owner_kind, cluster) (kube_pod_owner{owner_kind!=\"Job\"})  )) > 0",
+                      "expression": "sum by (namespace, controller, cluster) (max by(namespace, pod, cluster) (kube_pod_status_phase{job=\"kube-state-metrics\", phase=~\"Pending|Unknown\"}  ) * on(namespace, pod, cluster) group_left(controller)label_replace(kube_pod_owner ,\"controller\",\"$1\",\"owner_name\",\"(.*)\"))",
```

Comment:
missing >0

Reply:
Thanks for catching this. Added it back.